src/command.ts

```diff
@@ -195,53 +192,91 @@ function prepareOptions(
 
 /** Take a screenshot and move screenshot to base or actual folder */
 function takeScreenshot(
-  subject: string | undefined,
+  subject: JQueryWithSelector<HTMLElement> | void,
   name: string,
   screenshotOptions?: ScreenshotOptions
 ): Cypress.Chainable<TakeScreenshotProps> {
+  // @ts-expect-error TODO fix
   const objToOperateOn = subject !== undefined ? cy.get(subject) : cy
-  let ScreenshotDetails: TakeScreenshotProps
+  let screenshotDetails: TakeScreenshotProps
   return (
     objToOperateOn
       .screenshot(name, {
         ...screenshotOptions,
+        log: false,
         onAfterScreenshot(_el, props) {
-          ScreenshotDetails = props
+          screenshotDetails = props
           screenshotOptions?.onAfterScreenshot?.(_el, props)
         }
       })
       // @ts-expect-error todo: fix
-      .then(() => ScreenshotDetails)
+      .then(() => screenshotDetails)
   )
 }
 
 /** Call the plugin to compare snapshot images and generate a diff */
-function compareScreenshots(options: VisualRegressionOptions): Cypress.Chainable {
-  // @ts-expect-error todo: fix this
-  return cy.task('compareSnapshots', options).then((results: VisualRegressionResult) => {
-    console.log('VisualRegressionResult')
-    console.groupCollapsed(
-      `%c     VisualRegressionResult `,
-      'color: #ede917; background: #091806; padding: 12px 6px; border-radius: 4px; width:100%;'
-    )
-    console.info('%c VisualRegressionResult:', 'color: #FFF615; background: #091806; padding: 6px;', results)
-    console.groupEnd()
-    if (results.error !== undefined && !options.failSilently) {
-      throw new Error(results.error)
+function compareScreenshots(
+  subject: JQueryWithSelector<HTMLElement> | void,
+  options: VisualRegressionOptions
+): Cypress.Chainable<VisualRegressionResult> {
+  return cy.task<VisualRegressionResult>('compareSnapshots', options, { log: false }).then((result) => {
+    const log = Cypress.log({
+      type: 'parent',
+      name: 'compareScreenshots',
+      displayName: 'compareScreenshots',
+      message: "captureMode: 'fullPage'",
+      consoleProps: () => {
+        return {
+          Options: options,
+          Result: result
+        }
+      }
+    })
+    if (subject != null) {
+      log.set('$el', subject)
+      log.set('message', subject.selector)
+      log.set('type', 'child')
+    } else if (options.screenshotOptions.capture !== undefined) {
+      log.set('message', `captureMode: ${options.screenshotOptions.capture}`)
+    }
```

Comment:
This part will make the compareSnapshot command be logged like this:
<img width="1244" alt="image" src="https://github.com/user-attachments/assets/08b57319-a525-4098-95fd-eec9314009ce">